code/modules/clothing/suits/armor.dm

```diff
@@ -644,3 +644,47 @@
 	fire = 40
 	acid = 50
 	wound = 30
+
+/obj/item/clothing/suit/armor/vest/military
+	name = "Crude chestplate"
+	desc = "It may look rough, rusty and battered, but it's also made out of junk and uncomfortable to wear."
+	icon_state = "military"
+	inhand_icon_state = "armor"
+	dog_fashion = null
+	armor_type = /datum/armor/military
+	allowed = list(
+		/obj/item/banner,
+		/obj/item/claymore/shortsword,
+		/obj/item/nullrod,
+		/obj/item/spear,
+	)
+
+/datum/armor/military
+	melee = 45
+	bullet = 25
+	laser = 20
+	energy = 25
+	bomb = 25
+	fire = 10
+	acid = 50
+	wound = 20
+
+/obj/item/clothing/suit/armor/riot/knight/warlord
+	name = "golden plate armor"
+	desc = "This bulky set of armor coated with a shiny layer of gold. it seems to almost reflect all light sources."
```

Comment:
"This bulky set of armor is coated with a shiny layer of gold. It seems to almost reflect all light sources"